src/main/java/de/tum/in/www1/artemis/repository/FeedbackRepository.java

```diff
@@ -15,4 +15,6 @@
 @Repository
 public interface FeedbackRepository extends JpaRepository<Feedback, Long> {
     List<Feedback> findByResult(Result result);
+
+    void deleteByResult(Result result);
```

Comment:
I think we don’t need this anymore, right? Then please delete it

Reply:
done